python/gtsam/tests/test_Triangulation.py

```diff
@@ -20,14 +21,16 @@
 from gtsam.utils.test_case import GtsamTestCase
 
 
-class TestVisualISAMExample(GtsamTestCase):
+UPRIGHT = Rot3.Ypr(-np.pi / 2, 0., -np.pi / 2)
+
+
+class TestTriangulationExample(GtsamTestCase):
     """ Tests for triangulation with shared and individual calibrations """
 
     def setUp(self):
         """ Set up two camera poses """
         # Looking along X-axis, 1 meter above ground plane (x-y)
-        upright = Rot3.Ypr(-np.pi / 2, 0., -np.pi / 2)
-        pose1 = Pose3(upright, Point3(0, 0, 1))
+        pose1 = Pose3(UPRIGHT, Point3(0, 0, 1))
```

Comment:
You can just define `self.upright` as a variable here. Then you can access it in any test in this class as `self.upright`.